pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -34,16 +49,35 @@ let
     riscv64 = "FV/RISCV_VIRT";
   };
 
+  OvmfPkKek1AppPrefix = "4e32566d-8e9e-4f52-81d3-5bb9715f9727";
+
+  debian-edk-src = fetchurl {
+    url = "http://deb.debian.org/debian/pool/main/e/edk2/edk2_2023.11-5.debian.tar.xz";
+    sha256 = "1yxlab4md30pxvjadr6b4xn6cyfw0c292q63pyfv4vylvhsb24g4";
+  };
+
+  buildPrefix = "Build/*/*";
+
 in
 
+assert systemManagementModeRequired -> stdenv.hostPlatform.isx86;
+assert msVarsTemplate -> fdSize4MB;
+# TODO: Support other platforms.
```

Comment:
Are you unable to test this for aarch64? It would be great to keep full support for both major platforms.

riscv is a bit trickier due to the lack of binary cache, so I'm fine leaving that to the users of that platform to add.

Reply:
@adamcstephens I’m not sure how I’d test this. Is there easy virtualization support?

Reply:
If you don't have an arm device, you can try using binfmt to virtualize the platform. https://nixos.wiki/wiki/NixOS_on_ARM#Compiling_through_binfmt_QEMU

Reply:
Added aarch64 support. I’ve tested that OVMFFull builds, but not tried to use it.